packages/vscode-extension/src/webview/views/DeviceLocationView.tsx

```diff
@@ -0,0 +1,108 @@
+import { useProject } from "../providers/ProjectProvider";
+import React, { useState, useEffect } from "react";
+
+import "./DeviceLocationView.css";
+import Button from "../components/shared/Button";
+import Label from "../components/shared/Label";
+
+export function DeviceLocationView() {
+  const { project, deviceSettings } = useProject();
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    // @ts-ignore
+    const latitude = event.target[0].value;
+    // @ts-ignore
+    const longitude = event.target[2].value;
+    // @ts-ignore
+    const latDirection = event.target[1].value;
+    // @ts-ignore
+    const lonDirection = event.target[3].value;
```

Comment:
```suggestion
const [latitude, latDirection, longitude, lonDirection] = event.target.map(item => item.value);
```

the `.target[0]`, `.target[1]`, `.target[2]`... screams array destructuring.

here it's not a trivial example but I believe it is a bit readable? we could split it to 2 lines as well something like that
```js
  const locationValues = event.target.map(item => item.value);
  const [latitude, latDirection, longitude, lonDirection] = locationValues;
```